### PR TITLE
QA and upgrade to Spring Boot 2.4

### PR DESCRIPTION
**Pull request by mraible**

### CODE REVIEW

File: service-two/src/main/resources/bootstrap.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-spring.application.name=service-two
+spring.application.name=service-one
```
Comment (cavazosjoe) [inline service-two/src/main/resources/bootstrap.properties]: Typo?

File: service-two/pom.xml [hunk 2]
```diff
@@ -1,85 +1,93 @@
...
+	<name>service-two</name>
+	<description>Microservice Two</description>
 
-    <groupId>com.okta.dev.springcloudconfig</groupId>
-    <artifactId>service-two</artifactId>
-    <name>service-two</name>
-    <description>Service Two</description>
+	<properties>
+		<java.version>11</java.version>
+		<spring-cloud.version>2020.0.0-M6</spring-cloud.version>
```
Comment (cavazosjoe) [inline service-two/pom.xml:19]: Is the new version necessary to work with Spring Boot 2.4?

**Review by cavazosjoe — CHANGES_REQUESTED** [review]

File: service-two/src/main/resources/bootstrap.properties (hunk 1 above)
Comment (mraible) [reply service-two/src/main/resources/bootstrap.properties]: Yep! Good catch.

File: service-two/pom.xml (hunk 2 above)
Comment (mraible) [reply service-two/pom.xml:19]: This is what start.spring.io generates. It might be.

**Review by cavazosjoe — APPROVED** [review]

**Pull request merged**